config/source/multi/deployments/controller.yaml

```diff
@@ -47,12 +47,20 @@ spec:
 
         # How often (in seconds) the autoscaler tries to scale down the statefulset.
         - name: AUTOSCALER_REFRESH_PERIOD
-          value: '10'
+          value: '100'
 
         # The number of virtual replicas this pod can handle.
         - name: POD_CAPACITY
           value: '100'
 
+        # The number of zones in a multi-zone cluster, for spreading vreplica placement with HA
+        - name: SCHEDULE_SPREAD_ZONES
```

Comment:
What about `SCHEDULER_SPREAD_ZONES_COUNT`? At least `SCHEDULER` to match `AUTOSCALER` above